src/data_info/services/metadata.clj

```diff
@@ -285,6 +289,42 @@
   [data-id {:keys [user]} {:keys [dest recursive]}]
   (metadata-save user (uuidify data-id) (ft/rm-last-slash dest) (boolean recursive)))
 
+(defn- build-ore-uri
+  "Creates a URI for an ORE aggregation from an iRODS path."
+  [path]
+  (str (curl/url (cfg/commons-base) "browse" (string/replace path #"^/+" ""))))
```

Comment:
Removing the leading slashes from the path is necessary to prevent the base URL from being truncated.